Movie list

kgornostaeva opened the pull request:
Создание экрана со списком фильмов (коммит 003f2c9)
Для проверки использования diffUtil список с фильмами можно однократно обновить, потянув его вниз (используется SwipeRefreshLayout со SwipeRefreshListener)

kgornostaeva committed:
Movie details card

kgornostaeva committed:
Create a window with a list of movies

kgornostaeva committed:
Merge branch 'develop' into movie_list

NikolayUkhvarov commented on app/src/main/res/layout-land/item_movie.xml:
```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
```
снова отступы по 3 5 9 с:
повторяющиеся отступы в dimens, туда же размеры текста
(в других лэйаутах тоже)

kgornostaeva replied:
Коммитом f6a8a36 обновила отступы, некоторые округлила, большинство сохранила в dimens.
Также обновила звездочки в рейтинг баре, чтобы они корректно отображались.

NikolayUkhvarov commented on app/src/main/java/com/example/myapplication/MoviesViewHolder.kt:
```diff
@@ -0,0 +1,43 @@
+	}
+
+	override fun onClick(view: View?) {
+		val position: Int = this.layoutPosition
```
попробуй убрать логику из вьюхолдера

kgornostaeva replied:
Коммитом  f6a8a36 перенесла логику в адаптер. Также обновила отображение тостов, чтобы они не запускались, пока висит предыдущий тост.

kgornostaeva committed:
Move onClick logic from MoviesViewHolder to MoviesAdapter.
Update toasts displaying.
Update stars for movie list's rating bar.
Save dimensions in dimens and rounded up some of them.

kgornostaeva committed:
Merge branch 'movie_list' of https://github.com/kate-eversunny/android into movie_list

 Conflicts:
	app/src/main/res/values/dimen.xml
